packages/flame/lib/src/components/input/joystick_component.dart

```diff
@@ -52,6 +52,7 @@ class JoystickComponent extends HudMarginComponent with Draggable {
     double? size,
     double? knobRadius,
     Anchor anchor = Anchor.center,
+    int? priority,
```

Comment:
Is there a reason we prefer to add more and more arguments in the constructor, instead of letting the user to set them explicitly if they need to?
```dart
JoystickComponent() ..priority = 100;
```
(obviously, the setter for priority needs to be added, but that's quite easy).

Reply:
We have a set of arguments that we add to all components deriving from `PositionComponent` to keep consistency, the rest can be added with cascade notation.

Reply:
Ok, thanks.